index.js

```diff
@@ -602,6 +635,81 @@ function packAddress(derivationPath, xpub, hdPassphrase, derivationScheme) {
   )
 }
 
+function getAddressHeader(addressType, networkId) {
+  return Buffer.from([(addressType << 4) | networkId])
+}
+
+function getSpendingPart(pubKey) {
+  return blake2b(pubKey, 28)
+}
+
+function deriveBaseAddress(pubKey, stakePubKey, addressType, networkId, isStakeHash = false) {
```

Comment:
this is not technically a derivation as the function already receives the key. To be consistent with the existing API, I would prefix those functions with the "pack" verb instead of "derive"